stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -403,34 +403,42 @@ internal class ChannelControllerImpl(
 
     private fun loadMoreMessagesRequest(
         limit: Int = 30,
-        direction: Pagination
+        direction: Pagination,
+        messageId: String? = null,
```

Comment:
Took me a moment what this parameter meant, would be a good one to document briefly (can be used to specify the message we want to load relative to, otherwise the already loaded messages will be used in the query - something like that)

Same kind of doc could be added to the two methods calling this method, as they also have this slightly obscure parameter